analyzers/gradle/gradle.go

```diff
@@ -31,10 +32,11 @@ type Analyzer struct {
 }
 
 type Options struct {
-	Cmd           string `mapstructure:"cmd"`
-	Task          string `mapstructure:"task"`
-	Online        bool   `mapstructure:"online"`
-	AllSubmodules bool   `mapstructure:"allsubmodules"`
+	Cmd               string `mapstructure:"cmd"`
+	Task              string `mapstructure:"task"`
+	Online            bool   `mapstructure:"online"`
+	AllSubmodules     bool   `mapstructure:"allsubmodules"`
+	AllConfigurations bool   `mapstructure:"all-configurations"`
```

Comment:
[nit] I'd like these options to be consistent: either `all-submodules` + `all-configurations` or `allsubmodules` + `allconfigurations`.

For the sake of backwards compatibility, it's probably easier to make it the latter.

Reply:
Agreed... this was a mistake I made with `allmodules`.